src/sage/rings/function_field/khuri_makdisi.pyx

```diff
@@ -871,13 +871,31 @@ cdef class KhuriMakdisi_small(KhuriMakdisi_base):
             sage: p2 = G.point(pl2 - b)
             sage: -(-p1) == p1  # indirect doctest
             True
+
+        Check that :issue:`39148` is fixed::
+
+            sage: # long time
+            sage: k.<x> = FunctionField(GF(17)); t = polygen(k)
+            sage: F.<y> = k.extension(t^4 + (14*x + 14)*t^3 + 9*t^2 + (10*x^2 + 15*x + 8)*t
+            ....:  + 7*x^3 + 15*x^2 + 6*x + 16)
+            sage: infty1, infty2 = F.places_infinite()
+            sage: O = F.maximal_order()
+            sage: P = O.ideal((x + 1, y + 7)).divisor()
+            sage: D1 = 3*infty2 + infty1 - 4*P
+            sage: D2 = F.divisor_group().zero()
+            sage: J = F.jacobian(model='km-small', base_div=4*P)
+            sage: J(D1) + J(D2) == J(D1)
```

Comment:
```suggestion
            sage: J(D1) + J(D2) == J(D1)  # long time
```
I think this should get rid of the warning (and correctly mark this as a slow test)

Reply:
No. The block tag `sage: # long time` at line 877 already marks this line as a slow test and correctly skipped. But doctesting in github ci runs with `--long --warn-long 30.0`. Hence we get the slow test warning anyway even with `# long time` tag.

Reply:
Good to know. Everything is good then.